CHANGES.rst

```diff
@@ -5042,8 +5042,8 @@ Features added
 
 * ``LATEXMKOPTS`` variable for the Makefile in ``$BUILDDIR/latex`` to pass
   options to ``latexmk`` when executing ``make latexpdf`` (refs #3695, #3720)
-* Add a new event `env-check-consistency` to check consistency to extensions
-* Add `Domain.check_consistency()` to check consistency
+* Add a new event ``env-check-consistency`` to check consistency to extensions
+* Add ``Domain.check_consistency()`` to check consistency
```

Comment:
I would use the corresponding reference (if it's easy to use).

Reply:
If you want to go through and add commit suggestions thats fine,
but I would say that is not part of the scope of this PR and should be done in a separate PR

Reply:
Let's do it in a separate PR then. It's not that urgent

Reply:
cheers 😄